index.html

```diff
@@ -10,22 +10,150 @@
+							<li>
+								<a href="#"> Our Value</a>
+							</li>
+							<li>
+								<a href="#"> Events</a>
+							</li>
+							<li>
+								<a href="#"> Our Team</a>
+							</li>
+							<li>
+								<a href="#"> Contact</a>
+							</li>
+						</ul>
+					</div>
+					<a href="#" class="button2"> Support Us</a>
+					<div id="menu-container">
+						<button id="mobile-menu">
+							<svg
+								id="mobileOpen"
+								width="24"
+								height="19"
+								viewBox="0 0 24 19"
+								xmlns="http://www.w3.org/2000/svg"
+							>
+								<path
+									d="M22.3465 0H1.65354C1.215 0 0.794411 0.174212 0.484311 0.484311C0.174212 0.794411 0 1.215 0 1.65354C0 2.09209 0.174212 2.51267 0.484311 2.82277C0.794411 3.13287 1.215 3.30709 1.65354 3.30709H22.3465C22.785 3.30709 23.2056 3.13287 23.5157 2.82277C23.8258 2.51267 24 2.09209 24 1.65354C24 1.215 23.8258 0.794411 23.5157 0.484311C23.2056 0.174212 22.785 0 22.3465 0Z"
+								/>
+								<path
+									d="M22.3465 7.79492H1.65354C1.215 7.79492 0.794411 7.96913 0.484311 8.27923C0.174212 8.58933 0 9.00992 0 9.44847C0 9.88701 0.174212 10.3076 0.484311 10.6177C0.794411 10.9278 1.215 11.102 1.65354 11.102H22.3465C22.785 11.102 23.2056 10.9278 23.5157 10.6177C23.8258 10.3076 24 9.88701 24 9.44847C24 9.00992 23.8258 8.58933 23.5157 8.27923C23.2056 7.96913 22.785 7.79492 22.3465 7.79492Z"
+								/>
+								<path
+									d="M24 17.2444C24 16.8058 23.8258 16.3852 23.5157 16.0751C23.2056 15.765 22.785 15.5908 22.3465 15.5908H1.65354C1.215 15.5908 0.794411 15.765 0.484311 16.0751C0.174212 16.3852 0 16.8058 0 17.2444C0 17.6829 0.174212 18.1035 0.484311 18.4136C0.794411 18.7237 1.215 18.8979 1.65354 18.8979H22.3465C22.785 18.8979 23.2056 18.7237 23.5157 18.4136C23.8258 18.1035 24 17.6829 24 17.2444Z"
+								/>
+							</svg>
+							<svg
+								id="mobileClose"
+								class="hidden"
+								width="24"
+								height="24"
+								viewBox="0 0 24 24"
+								fill="none"
+								xmlns="http://www.w3.org/2000/svg"
+							>
+								<g id="Group">
+									<g id="Group_2">
+										<path
+											id="Vector"
+											d="M1.7416 23.9993C1.29575 23.9993 0.849903 23.8293 0.509942 23.4894C-0.169981 22.8094 -0.169981 21.7067 0.509942 21.026L21.0267 0.509942C21.7067 -0.169981 22.8094 -0.169981 23.4901 0.509942C24.17 1.18986 24.17 2.29265 23.4901 2.97327L2.97257 23.4894C2.63261 23.8293 2.18745 23.9993 1.7416 23.9993Z"
+											fill="white"
+										/>
+									</g>
+									<g id="Group_3">
+										<path
+											id="Vector_2"
+											d="M22.2577 24C21.8118 24 21.366 23.83 21.026 23.4901L0.509942 2.97327C-0.169981 2.29334 -0.169981 1.19056 0.509942 0.509942C1.18986 -0.169981 2.29265 -0.169981 2.97327 0.509942L23.4901 21.0267C24.17 21.7067 24.17 22.8094 23.4901 23.4901C23.1494 23.83 22.7035 24 22.2577 24Z"
+											fill="white"
+										/>
+									</g>
+								</g>
+							</svg>
+						</button>
+	
+						<div id="nav-links-container" class="hidden">
```

Comment:
Ideally, we would have just one menu and change the display with CSS. Let me know if you want me to help you with this tomorrow.

Reply:
If not, we can just add the links here as well :)